torch/distributions.py

```diff
@@ -201,12 +201,16 @@ def log_prob(self, value):
 class _StandardGamma(Function):
     @staticmethod
     def forward(ctx, alpha):
-        ctx.saved_grad = alpha.new(alpha.size())
-        return torch._C._standard_gamma_with_grad(alpha, ctx.saved_grad)
+        ctx.save_for_backward(alpha)
+        ctx.saved_u = alpha.new(alpha.size())
+        ctx.saved_x = alpha.new(alpha.size())
+        return torch._C._standard_gamma_fwd(alpha, ctx.saved_u, ctx.saved_x)
 
     @staticmethod
     def backward(ctx, grad_output):
-        return grad_output * Variable(ctx.saved_grad)
+        alpha, = ctx.saved_variables
```

Comment:
This backward isn't really differentiable twice. Can you mark it `@once_differentiable` and use `ctx.saved_tensors`? `grad_output` will become a tensor too.